packages/api/cms-api/src/dam/files/files.utils.ts

```diff
@@ -94,3 +104,31 @@ export const getValidExtensionsForMimetype = (mimetype: string) => {
 
     return supportedExtensions;
 };
+
+export async function createFileUploadInputFromUrl(url: string): Promise<FileUploadInput> {
```

Comment:
There's no need for it to be a service since it doesn't require Nest specifics. Also FileUploadService is too similar to FileUploadsService. See https://github.com/vivid-planet/comet/pull/2336#discussion_r1691106859.